assets/css/index.css

```diff
@@ -6,17 +6,27 @@
   align-items: flex-start;
 }
 
-#instafeed a {
-  border-bottom: none;
-  padding: none;
-  height: 150px;
-}
+  #instafeed a {
+    border-bottom: none;
+    padding: none;
+    height: 150px;
+  }
```

Comment:
Can you give this page some formatting love? I don't think you meant to indent this.